jest.config.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+/** @type {import('jest').Config} */
+const config = {
```

Comment:
do you have any customized settings? if not, maybe no need this file?

Reply:
In the jest.config.js, I do the following setting:

- Using jest-expect-message package to write custom message

```javascript
{
  setupFilesAfterEnv: ['jest-expect-message'],
}
```

- Specify the file which should be executed by jest (file with *.spec.js)
  - If not specified, test/test.js will also be executed by default

```javascript
{
  testMatch: [
    "**/test/**/*.spec.js"
  ],
}
```